docker/ubuntu1604/Dockerfile

```diff
@@ -52,8 +52,9 @@ RUN echo "deb http://apt.openvstorage.com unstable main" > /etc/apt/sources.list
     	    libxio0 libxio0-dbg libxio0-dbgsym \
     	    libxio-dev libxio-dev-dbgsym && \
     apt-get install -y libev4 && \
-    cd /root && apt-get download --allow-unauthenticated arakoon && \
+    cd /tmp && apt-get download --allow-unauthenticated arakoon && \
```

Comment:
@dejonghb , there is a bug in Xenial (ubuntu 16.04 ) permissions related and using root as a download directory does not work.
id you do not like tmp , it would be better to allocate some other tmp directory.